app/scripts/api/app.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+require('angular');
+
+angular.module('Liberator.api', [
+    require('angular-ui-router'),
```

Comment:
So far, the convention for javascript is 2 spaces, setup your editor to work with 2 spaces in javascript